.studio/automate-ui

```diff
@@ -259,7 +259,7 @@ function _install_nodejs_from_nvmrc() {
   fi
 
   log_line "Installing NodeJS (v.$NODE_VERSION)"
-  OPTS="-b" install "core/node10/$NODE_VERSION" >/dev/null
+  OPTS="-b" install "core/node/$NODE_VERSION" >/dev/null
```

Comment:
Two previous PRs (#785 and #2267) updated plan.sh
in a couple projects to be specific to a node.js major version.
I now believe that was an unnecessary, and in fact, brittle change.
A node.js package newly installed in core-plans makes its way
to the "main branch" (https://bldr.habitat.sh/#/pkgs/core/node).
References should remain invariant to that location
so they do not need updating when node.js major versions change.